### DISCUSSION TITLE
There are some shadow effect issues

### DISCUSSION THREAD

**Discussion by fuchao** [google/filament#7453] [Q&A] [answered]
Filament:1.49.0
1、When I turn on the shadows in Direct light, there may be ripples on the model at certain times
![50aa95d050dbd7e800ad09e887f1175](https://github.com/google/filament/assets/1783838/c59d6c04-e6c1-407c-95fe-1e71dfd5c72b)

2、When I open SSAO, use the default settings. The following situations may occur
![1703249384126](https://github.com/google/filament/assets/1783838/baee3082-a52b-4a7b-bfc8-bf27348a4544)
![1703249512058](https://github.com/google/filament/assets/1783838/8711e68e-b122-4a3b-9e74-b88653304b3e)

Do I need any other settings

**Comment by romainguy** [comment, marked as answer]
The first screenshot shows acne. You need to play with the shadow bias/offset in the shadow map options of the light.
